archetype/engine-v2/src/test/resources/schema/archetype.xsd

```diff
@@ -0,0 +1,1149 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you plan on using this file as part of the tests ? If not, let's avoid duplicating it.

Reply:
Yes it is the file the reader is parsing for the test. Would you like me to modify the content to make it shorter maybe and use more complex structure for testing purpose like recursive list etc for example ?

Reply:
You are reading the schema ?

Reply:
Yes in `ArchetypeDescriptorTest#testUnmarshall` I read the schema. I copied on the V1 test.